PlayerControls/sources/AdVideoControls.swift

```diff
@@ -75,12 +75,12 @@ public final class AdVideoControls: UIViewController {
         public let isLoading: Bool
         public let airplayActiveViewHidden: Bool
         
-        public enum ClickAction: Prism {
+        public enum ClickAction: Prism, AutoCodable {
```

Comment:
Maybe better to create "AutoGeneratable" protocol?

Reply:
`AutoGeneratable` is a too abstract name, as for me. But, i'm agreed that with one protocol code will be cleaner. Need @AlexeyDemedetskiy and @AndriiMoskvin opinion

Reply:
`AutoGeneratable` can be parent for `Prism` and `AutoCodable`

Reply:
Sounds good)

Reply:
@BogdanBilonog I've discussed this case with @AlexeyDemedetskiy and he proposed another solution. 
Solution is:
To create a special file with protocol declaration and extension for type witch suppose to confirm this protocol. (Same as `Defaultable`)

Reply:
@AndriiDoroshko ok